scripts/firedrake-zenodo

```diff
@@ -235,13 +235,18 @@ tag += "." + str(index + 1)
 for component in components:
     repo = gh.repository(projects[component], component)
 
-    commit = repo.commit(shas[component])
-    if not commit:
-        log.error("Failed to find specified commit for %s" % component)
-    shas[component] = commit.sha
+    try:
+        commit = repo.commit(shas[component])
+        if not commit:
+            log.error("Failed to find specified commit for %s" % component)
```

Comment:
I realise this is not changed from previously, but should this be an error that causes the script to exit?